catalog/app/containers/Bucket/PackageList.js

```diff
@@ -103,7 +103,7 @@ export default ({
                         <Link to={urls.bucketPackageDetail(bucket, name)}>{name}</Link>
                       </Typography>
                       <Typography variant="body1">
-                        Updated on {modified.toLocaleString()}
+                        Updated on {modified ? modified.toLocaleString() : '???'}
```

Comment:
Can you make this `"[unknown: missing latest in .quilt]"` or something like that?